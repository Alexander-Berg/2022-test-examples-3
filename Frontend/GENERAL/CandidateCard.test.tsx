import React from 'react';
import renderer from 'react-test-renderer';

import { CONSIDERATION_ISSUE_LEVEL } from '~/enums/ConsiderationIssueLevel';

import {
    CandidateCard, ICandidateCardProps,
} from '.';

const baseCard = {
    id: 1,
    extended_status: 'hr_screening_finished',
    extended_status_changed_at: '2018-09-13T10:26:24.534475Z',
    resolution: '',
    state: 'in_progress',
};

const baseCandidate = {
    first_name: 'Ольга',
    last_name: 'Козлова',
    id: 1,
    is_current_employee: false,
};

describe('Jest snapshot тесты на компонент CandidateCard', () => {
    it('Кандидат без логина', () => {
        const card: ICandidateCardProps = {
            ...baseCard,
            // @ts-ignore
            candidate: baseCandidate,
        };
        const CandidateCardComponent = renderer.create(
            <CandidateCard
                {...card}
            />).toJSON();
        expect(CandidateCardComponent).toMatchSnapshot();
    });

    it('Кандидат действующий сотрудник', () => {
        const card: ICandidateCardProps = {
            ...baseCard,
            // @ts-ignore
            candidate: {
                ...baseCandidate,
                login: 'olgakozlova',
                is_current_employee: true,
            },
        };
        const CandidateCardComponent = renderer.create(
            <CandidateCard
                {...card}
            />).toJSON();
        expect(CandidateCardComponent).toMatchSnapshot();
    });

    it('Кандидат бывший сотрудник', () => {
        const card: ICandidateCardProps = {
            ...baseCard,
            // @ts-ignore
            candidate: {
                login: 'olgakozlova',
                ...baseCandidate,
            },
        };
        const CandidateCardComponent = renderer.create(
            <CandidateCard
                {...card}
            />).toJSON();
        expect(CandidateCardComponent).toMatchSnapshot();
    });

    it('Кандидат с профессиями < 2', () => {
        const card: ICandidateCardProps = {
            ...baseCard,
            // @ts-ignore
            candidate: {
                ...baseCandidate,
                professions: [
                    {
                        id: 1,
                        name: 'Разработчик бекэнда',
                    },
                ],
            },
        };
        const CandidateCardComponent = renderer.create(
            <CandidateCard
                {...card}
            />).toJSON();
        expect(CandidateCardComponent).toMatchSnapshot();
    });
    it('Кандидат с профессиями >= 2', () => {
        const card: ICandidateCardProps = {
            ...baseCard,
            // @ts-ignore
            candidate: {
                ...baseCandidate,
                professions: [
                    {
                        id: 1,
                        name: 'Разработчик бекэнда',
                    },
                    {
                        id: 2,
                        name: 'Разработчик фронтенда',
                    },
                ],
            },
        };
        const CandidateCardComponent = renderer.create(
            <CandidateCard
                {...card}
            />).toJSON();
        expect(CandidateCardComponent).toMatchSnapshot();
    });
    it('Кандидат с городами < 2', () => {
        const card: ICandidateCardProps = {
            ...baseCard,
            // @ts-ignore
            candidate: {
                ...baseCandidate,
                target_cities: [
                    {
                        id: 1,
                        name: 'Минск',
                    },
                ],
            },
        };
        const CandidateCardComponent = renderer.create(
            <CandidateCard
                {...card}
            />).toJSON();
        expect(CandidateCardComponent).toMatchSnapshot();
    });
    it('Кандидат с городами >= 2', () => {
        const card: ICandidateCardProps = {
            // @ts-ignore
            candidate: {
                ...baseCandidate,
                target_cities: [
                    {
                        id: 1,
                        name: 'Минск',
                    },
                    {
                        id: 2,
                        name: 'Брест',
                    },
                ],
            },
            extended_status_changed_at: '2021-04-23T17:42:34.360023Z',
        };
        const CandidateCardComponent = renderer.create(
            <CandidateCard
                {...card}
            />).toJSON();
        expect(CandidateCardComponent).toMatchSnapshot();
    });
    it('Кандидат только со скринингами', () => {
        const card: ICandidateCardProps = {
            // @ts-ignore
            candidate: { ...baseCandidate },
            interviews: [
                {
                    grade: null,
                    id: 31632,
                    resolution: 'nohire',
                    state: 'finished',
                    type: 'hr_screening',
                    yandex_grade: null,
                },
                {
                    grade: null,
                    id: 31634,
                    resolution: 'hire',
                    state: 'finished',
                    type: 'hr_screening',
                    yandex_grade: null,
                },
            ],
            extended_status_changed_at: '2021-04-23T17:42:34.360023Z',
        };
        const CandidateCardComponent = renderer.create(
            <CandidateCard
                {...card}
            />).toJSON();
        expect(CandidateCardComponent).toMatchSnapshot();
    });
    it('Кандидат с прочими видами интервью', () => {
        const card: ICandidateCardProps = {
            // @ts-ignore
            candidate: { ...baseCandidate },
            interviews: [
                {
                    grade: null,
                    id: 31632,
                    resolution: 'nohire',
                    state: 'finished',
                    type: 'hr_screening',
                    yandex_grade: null,
                },
                {
                    id: 33840,
                    type: 'screening',
                    grade: null,
                    state: 'assigned',
                    resolution: '',
                    yandex_grade: null,
                },
                {
                    grade: 0,
                    id: 31036,
                    resolution: '',
                    state: 'estimated',
                    type: 'screening',
                    yandex_grade: null,
                },
                {
                    id: 32751,
                    type: 'screening',
                    grade: 3,
                    state: 'finished',
                    resolution: '',
                    yandex_grade: null,
                },
                {
                    id: 31907,
                    type: 'aa',
                    grade: 4,
                    state: 'finished',
                    resolution: '',
                    yandex_grade: null,
                },
                {
                    grade: 0,
                    id: 31038,
                    resolution: '',
                    state: 'finished',
                    type: 'aa',
                    yandex_grade: null,
                },
                {
                    id: 319407,
                    type: 'aa',
                    grade: null,
                    state: 'assigned',
                    resolution: '',
                    yandex_grade: null,
                },
                {
                    id: 31210,
                    type: 'regular',
                    grade: 2,
                    state: 'finished',
                    resolution: '',
                    yandex_grade: null,
                },
                {
                    id: 321210,
                    type: 'regular',
                    grade: 0,
                    state: 'finished',
                    resolution: '',
                    yandex_grade: null,
                },
                {
                    id: 326210,
                    type: 'regular',
                    grade: null,
                    state: 'assigned',
                    resolution: '',
                    yandex_grade: null,
                },
                {
                    id: 31944,
                    type: 'final',
                    grade: null,
                    state: 'finished',
                    resolution: 'hire',
                    yandex_grade: null,
                },
                {
                    id: 319444,
                    type: 'final',
                    grade: null,
                    state: 'finished',
                    resolution: 'nohire',
                    yandex_grade: null,
                },
                {
                    id: 319444,
                    type: 'final',
                    grade: null,
                    state: 'assigned',
                    resolution: '',
                    yandex_grade: null,
                },
            ],
            extended_status_changed_at: '2021-04-23T17:42:34.360023Z',
        };
        const CandidateCardComponent = renderer.create(
            <CandidateCard
                {...card}
            />).toJSON();
        expect(CandidateCardComponent).toMatchSnapshot();
    });
    it('Кандидат c issue на дату изменения статуса', () => {
        const card: ICandidateCardProps = {
            // @ts-ignore
            candidate: {
                ...baseCandidate,
                issues: [
                    {
                        id: 1,
                        type: 'extended_status_changed_at',
                        level: CONSIDERATION_ISSUE_LEVEL.DANGER,
                        params: {},
                    },
                ],
            },
            extended_status_changed_at: '2021-04-23T17:42:34.360023Z',
        };
        const CandidateCardComponent = renderer.create(
            <CandidateCard
                {...card}
            />).toJSON();
        expect(CandidateCardComponent).toMatchSnapshot();
    });
});
