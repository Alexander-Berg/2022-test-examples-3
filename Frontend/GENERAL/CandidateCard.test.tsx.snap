// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest snapshot тесты на компонент CandidateCard Кандидат c issue на дату изменения статуса 1`] = `
<div
  className="BoardCard CandidateCard"
>
  <a
    className="Link BoardCard-Link"
    href="/candidates/1/"
    target="_blank"
  >
    <div
      className="BoardCard-Row"
    >
      <div
        className="BoardCard-RowItem BoardCard-RowItem_isHeading"
      >
        <div
          className="CandidateName CandidateCard-Name"
        >
          <a
            className="Link Link_theme_black CandidateName-Name"
            href="/candidates/1/"
            target="_blank"
          >
            Ольга Козлова
          </a>
           
        </div>
      </div>
      <div
        className="BoardCard-RowItem"
      >
        <span
          className="Id Id_theme_ghost CandidateCard-Id"
        >
          CAN 
          1
        </span>
      </div>
    </div>
    <div
      className="BoardCard-Row"
    >
      Статус изменён
       
      <span
        className="DisplayDate"
        title="23 апреля 2021, 20:42"
      >
        23 апреля 2021
      </span>
    </div>
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
  </a>
</div>
`;

exports[`Jest snapshot тесты на компонент CandidateCard Кандидат без логина 1`] = `
<div
  className="BoardCard CandidateCard"
>
  <a
    className="Link BoardCard-Link"
    href="/candidates/1/"
    target="_blank"
  >
    <div
      className="BoardCard-Row"
    >
      <div
        className="BoardCard-RowItem BoardCard-RowItem_isHeading"
      >
        <div
          className="CandidateName CandidateCard-Name"
        >
          <a
            className="Link Link_theme_black CandidateName-Name"
            href="/candidates/1/"
            target="_blank"
          >
            Ольга Козлова
          </a>
           
        </div>
      </div>
      <div
        className="BoardCard-RowItem"
      >
        <span
          className="Id Id_theme_ghost CandidateCard-Id"
        >
          CAN 
          1
        </span>
      </div>
    </div>
    <div
      className="BoardCard-Row"
    >
      Статус изменён
       
      <span
        className="DisplayDate"
        title="13 сентября 2018, 13:26"
      >
        13 сентября 2018
      </span>
    </div>
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
  </a>
</div>
`;

exports[`Jest snapshot тесты на компонент CandidateCard Кандидат бывший сотрудник 1`] = `
<div
  className="BoardCard CandidateCard"
>
  <a
    className="Link BoardCard-Link"
    href="/candidates/1/"
    target="_blank"
  >
    <div
      className="BoardCard-Row"
    >
      <div
        className="BoardCard-RowItem BoardCard-RowItem_isHeading"
      >
        <div
          className="CandidateName CandidateCard-Name"
        >
          <a
            className="Link Link_theme_black CandidateName-Name"
            href="/candidates/1/"
            target="_blank"
          >
            Ольга Козлова
          </a>
           
          <span>
            <a
              className="Link StaffUser StaffUser_isLink StaffUser_isDismissed"
              href="https://staff.yandex-team.ru/olgakozlova"
              target="_blank"
            >
              <span
                className="StaffUser-First"
              >
                @
              </span>
              olgakozlova
            </a>
          </span>
        </div>
      </div>
      <div
        className="BoardCard-RowItem"
      >
        <span
          className="Id Id_theme_ghost CandidateCard-Id"
        >
          CAN 
          1
        </span>
      </div>
    </div>
    <div
      className="BoardCard-Row"
    >
      Статус изменён
       
      <span
        className="DisplayDate"
        title="13 сентября 2018, 13:26"
      >
        13 сентября 2018
      </span>
    </div>
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
  </a>
</div>
`;

exports[`Jest snapshot тесты на компонент CandidateCard Кандидат действующий сотрудник 1`] = `
<div
  className="BoardCard CandidateCard"
>
  <a
    className="Link BoardCard-Link"
    href="/candidates/1/"
    target="_blank"
  >
    <div
      className="BoardCard-Row"
    >
      <div
        className="BoardCard-RowItem BoardCard-RowItem_isHeading"
      >
        <div
          className="CandidateName CandidateCard-Name"
        >
          <a
            className="Link Link_theme_black CandidateName-Name"
            href="/candidates/1/"
            target="_blank"
          >
            Ольга Козлова
          </a>
           
          <span>
            <a
              className="Link StaffUser StaffUser_isLink"
              href="https://staff.yandex-team.ru/olgakozlova"
              target="_blank"
            >
              <span
                className="StaffUser-First"
              >
                @
              </span>
              olgakozlova
            </a>
          </span>
        </div>
      </div>
      <div
        className="BoardCard-RowItem"
      >
        <span
          className="Id Id_theme_ghost CandidateCard-Id"
        >
          CAN 
          1
        </span>
      </div>
    </div>
    <div
      className="BoardCard-Row"
    >
      Статус изменён
       
      <span
        className="DisplayDate"
        title="13 сентября 2018, 13:26"
      >
        13 сентября 2018
      </span>
    </div>
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
  </a>
</div>
`;

exports[`Jest snapshot тесты на компонент CandidateCard Кандидат с городами < 2 1`] = `
<div
  className="BoardCard CandidateCard"
>
  <a
    className="Link BoardCard-Link"
    href="/candidates/1/"
    target="_blank"
  >
    <div
      className="BoardCard-Row"
    >
      <div
        className="BoardCard-RowItem BoardCard-RowItem_isHeading"
      >
        <div
          className="CandidateName CandidateCard-Name"
        >
          <a
            className="Link Link_theme_black CandidateName-Name"
            href="/candidates/1/"
            target="_blank"
          >
            Ольга Козлова
          </a>
           
        </div>
      </div>
      <div
        className="BoardCard-RowItem"
      >
        <span
          className="Id Id_theme_ghost CandidateCard-Id"
        >
          CAN 
          1
        </span>
      </div>
    </div>
    <div
      className="BoardCard-Row"
    >
      Статус изменён
       
      <span
        className="DisplayDate"
        title="13 сентября 2018, 13:26"
      >
        13 сентября 2018
      </span>
    </div>
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    >
      <div
        className="ListShorter CandidateCard-Cities"
      >
        <div
          className="ListShorter-List ListShorter-List_size_m"
        >
          <span
            className="ListShorter-ListItem"
          >
            Минск
          </span>
        </div>
      </div>
    </div>
  </a>
</div>
`;

exports[`Jest snapshot тесты на компонент CandidateCard Кандидат с городами >= 2 1`] = `
<div
  className="BoardCard CandidateCard"
>
  <a
    className="Link BoardCard-Link"
    href="/candidates/1/"
    target="_blank"
  >
    <div
      className="BoardCard-Row"
    >
      <div
        className="BoardCard-RowItem BoardCard-RowItem_isHeading"
      >
        <div
          className="CandidateName CandidateCard-Name"
        >
          <a
            className="Link Link_theme_black CandidateName-Name"
            href="/candidates/1/"
            target="_blank"
          >
            Ольга Козлова
          </a>
           
        </div>
      </div>
      <div
        className="BoardCard-RowItem"
      >
        <span
          className="Id Id_theme_ghost CandidateCard-Id"
        >
          CAN 
          1
        </span>
      </div>
    </div>
    <div
      className="BoardCard-Row"
    >
      Статус изменён
       
      <span
        className="DisplayDate"
        title="23 апреля 2021, 20:42"
      >
        23 апреля 2021
      </span>
    </div>
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    >
      <div
        className="ListShorter CandidateCard-Cities"
      >
        <div
          className="ListShorter-List ListShorter-List_size_m"
        >
          <span
            className="ListShorter-ListItem"
          >
            Минск
          </span>
          <span
            className="ListShorter-Hidden ListShorter-Hidden_placeMoreOnNewLine"
          >
            <span
              className="ListShorter-ListMore"
              title="Брест"
            >
              ещё +1
            </span>
          </span>
        </div>
      </div>
    </div>
  </a>
</div>
`;

exports[`Jest snapshot тесты на компонент CandidateCard Кандидат с профессиями < 2 1`] = `
<div
  className="BoardCard CandidateCard"
>
  <a
    className="Link BoardCard-Link"
    href="/candidates/1/"
    target="_blank"
  >
    <div
      className="BoardCard-Row"
    >
      <div
        className="BoardCard-RowItem BoardCard-RowItem_isHeading"
      >
        <div
          className="CandidateName CandidateCard-Name"
        >
          <a
            className="Link Link_theme_black CandidateName-Name"
            href="/candidates/1/"
            target="_blank"
          >
            Ольга Козлова
          </a>
           
        </div>
      </div>
      <div
        className="BoardCard-RowItem"
      >
        <span
          className="Id Id_theme_ghost CandidateCard-Id"
        >
          CAN 
          1
        </span>
      </div>
    </div>
    <div
      className="BoardCard-Row"
    >
      Статус изменён
       
      <span
        className="DisplayDate"
        title="13 сентября 2018, 13:26"
      >
        13 сентября 2018
      </span>
    </div>
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    >
      <div
        className="ListShorter CandidateCard-Professions"
      >
        <div
          className="ListShorter-List ListShorter-List_size_m"
        >
          <span
            className="ListShorter-ListItem"
          >
            Разработчик бекэнда
          </span>
        </div>
      </div>
    </div>
    <div
      className="BoardCard-Row"
    />
  </a>
</div>
`;

exports[`Jest snapshot тесты на компонент CandidateCard Кандидат с профессиями >= 2 1`] = `
<div
  className="BoardCard CandidateCard"
>
  <a
    className="Link BoardCard-Link"
    href="/candidates/1/"
    target="_blank"
  >
    <div
      className="BoardCard-Row"
    >
      <div
        className="BoardCard-RowItem BoardCard-RowItem_isHeading"
      >
        <div
          className="CandidateName CandidateCard-Name"
        >
          <a
            className="Link Link_theme_black CandidateName-Name"
            href="/candidates/1/"
            target="_blank"
          >
            Ольга Козлова
          </a>
           
        </div>
      </div>
      <div
        className="BoardCard-RowItem"
      >
        <span
          className="Id Id_theme_ghost CandidateCard-Id"
        >
          CAN 
          1
        </span>
      </div>
    </div>
    <div
      className="BoardCard-Row"
    >
      Статус изменён
       
      <span
        className="DisplayDate"
        title="13 сентября 2018, 13:26"
      >
        13 сентября 2018
      </span>
    </div>
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    >
      <div
        className="ListShorter CandidateCard-Professions"
      >
        <div
          className="ListShorter-List ListShorter-List_size_m"
        >
          <span
            className="ListShorter-ListItem"
          >
            Разработчик бекэнда
          </span>
          <span
            className="ListShorter-Hidden ListShorter-Hidden_placeMoreOnNewLine"
          >
            <span
              className="ListShorter-ListMore"
              title="Разработчик фронтенда"
            >
              ещё +1
            </span>
          </span>
        </div>
      </div>
    </div>
    <div
      className="BoardCard-Row"
    />
  </a>
</div>
`;

exports[`Jest snapshot тесты на компонент CandidateCard Кандидат с прочими видами интервью 1`] = `
<div
  className="BoardCard CandidateCard"
>
  <a
    className="Link BoardCard-Link"
    href="/candidates/1/"
    target="_blank"
  >
    <div
      className="BoardCard-Row"
    >
      <div
        className="BoardCard-RowItem BoardCard-RowItem_isHeading"
      >
        <div
          className="CandidateName CandidateCard-Name"
        >
          <a
            className="Link Link_theme_black CandidateName-Name"
            href="/candidates/1/"
            target="_blank"
          >
            Ольга Козлова
          </a>
           
        </div>
      </div>
      <div
        className="BoardCard-RowItem"
      >
        <span
          className="Id Id_theme_ghost CandidateCard-Id"
        >
          CAN 
          1
        </span>
      </div>
    </div>
    <div
      className="BoardCard-Row"
    >
      Статус изменён
       
      <span
        className="DisplayDate"
        title="23 апреля 2021, 20:42"
      >
        23 апреля 2021
      </span>
    </div>
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="CardInterviewsInfo"
    >
      <div
        className="CardInterviewsInfo-InterviewStat"
        title=""
      >
        <div
          className="InterviewType CardInterviewsInfo-InterviewType"
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_hr"
            style={Object {}}
          />
        </div>
        <div
          className="InterviewGrade InterviewGrade_result_rejected"
          title=""
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_no InterviewGrade-Icon InterviewGrade-Icon_result_rejected"
            style={Object {}}
          />
        </div>
      </div>
      <div
        className="CardInterviewsInfo-InterviewStat"
        title="Назначено 3 skype секции"
      >
        <div
          className="InterviewType CardInterviewsInfo-InterviewType"
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_video"
            style={Object {}}
          />
        </div>
        <div
          className="InterviewGrade InterviewGrade_result_unknown"
          title="Секция не завершена"
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_help InterviewGrade-Icon InterviewGrade-Icon_result_unknown"
            style={Object {}}
          />
        </div>
        <div
          className="InterviewGrade InterviewGrade_result_rejected"
          title=""
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_no InterviewGrade-Icon InterviewGrade-Icon_result_rejected"
            style={Object {}}
          />
        </div>
        <div
          className="InterviewGrade InterviewGrade_result_hired"
          title=""
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_three InterviewGrade-Icon InterviewGrade-Icon_result_hired"
            style={Object {}}
          />
        </div>
      </div>
      <div
        className="CardInterviewsInfo-InterviewStat"
        title="Назначено 3 секции с кодом"
      >
        <div
          className="InterviewType CardInterviewsInfo-InterviewType"
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_code"
            style={Object {}}
          />
        </div>
        <div
          className="InterviewGrade InterviewGrade_result_rejected"
          title=""
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_no InterviewGrade-Icon InterviewGrade-Icon_result_rejected"
            style={Object {}}
          />
        </div>
        <div
          className="InterviewGrade InterviewGrade_result_unknown"
          title="Секция не завершена"
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_help InterviewGrade-Icon InterviewGrade-Icon_result_unknown"
            style={Object {}}
          />
        </div>
        <div
          className="InterviewGrade InterviewGrade_result_hired"
          title=""
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_two InterviewGrade-Icon InterviewGrade-Icon_result_hired"
            style={Object {}}
          />
        </div>
      </div>
      <div
        className="CardInterviewsInfo-InterviewStat"
        title="Назначено 3 AA секции"
      >
        <div
          className="InterviewType CardInterviewsInfo-InterviewType"
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_aa"
            style={Object {}}
          />
        </div>
        <div
          className="InterviewGrade InterviewGrade_result_rejected"
          title=""
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_no InterviewGrade-Icon InterviewGrade-Icon_result_rejected"
            style={Object {}}
          />
        </div>
        <div
          className="InterviewGrade InterviewGrade_result_unknown"
          title="Секция не завершена"
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_help InterviewGrade-Icon InterviewGrade-Icon_result_unknown"
            style={Object {}}
          />
        </div>
        <div
          className="InterviewGrade InterviewGrade_result_hired"
          title=""
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_four InterviewGrade-Icon InterviewGrade-Icon_result_hired"
            style={Object {}}
          />
        </div>
      </div>
      <div
        className="CardInterviewsInfo-InterviewStat"
        title="Назначено 3 финальные секции"
      >
        <div
          className="InterviewType CardInterviewsInfo-InterviewType"
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_final"
            style={Object {}}
          />
        </div>
        <div
          className="InterviewGrade InterviewGrade_result_unknown"
          title="Секция не завершена"
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_help InterviewGrade-Icon InterviewGrade-Icon_result_unknown"
            style={Object {}}
          />
        </div>
        <div
          className="InterviewGrade InterviewGrade_result_hired"
          title=""
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_tick InterviewGrade-Icon InterviewGrade-Icon_result_hired"
            style={Object {}}
          />
        </div>
        <div
          className="InterviewGrade InterviewGrade_result_rejected"
          title=""
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_no InterviewGrade-Icon InterviewGrade-Icon_result_rejected"
            style={Object {}}
          />
        </div>
      </div>
    </div>
  </a>
</div>
`;

exports[`Jest snapshot тесты на компонент CandidateCard Кандидат только со скринингами 1`] = `
<div
  className="BoardCard CandidateCard"
>
  <a
    className="Link BoardCard-Link"
    href="/candidates/1/"
    target="_blank"
  >
    <div
      className="BoardCard-Row"
    >
      <div
        className="BoardCard-RowItem BoardCard-RowItem_isHeading"
      >
        <div
          className="CandidateName CandidateCard-Name"
        >
          <a
            className="Link Link_theme_black CandidateName-Name"
            href="/candidates/1/"
            target="_blank"
          >
            Ольга Козлова
          </a>
           
        </div>
      </div>
      <div
        className="BoardCard-RowItem"
      >
        <span
          className="Id Id_theme_ghost CandidateCard-Id"
        >
          CAN 
          1
        </span>
      </div>
    </div>
    <div
      className="BoardCard-Row"
    >
      Статус изменён
       
      <span
        className="DisplayDate"
        title="23 апреля 2021, 20:42"
      >
        23 апреля 2021
      </span>
    </div>
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="BoardCard-Row"
    />
    <div
      className="CardInterviewsInfo"
    >
      <div
        className="CardInterviewsInfo-InterviewStat"
        title=""
      >
        <div
          className="InterviewType CardInterviewsInfo-InterviewType"
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_hr"
            style={Object {}}
          />
        </div>
        <div
          className="InterviewGrade InterviewGrade_result_hired"
          title=""
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_tick InterviewGrade-Icon InterviewGrade-Icon_result_hired"
            style={Object {}}
          />
        </div>
        <div
          className="InterviewGrade InterviewGrade_result_rejected"
          title=""
        >
          <span
            aria-hidden={true}
            className="Icon Icon_size_s Icon_type_no InterviewGrade-Icon InterviewGrade-Icon_result_rejected"
            style={Object {}}
          />
        </div>
      </div>
    </div>
  </a>
</div>
`;
